css/mail.scss

```diff
@@ -5,11 +5,17 @@
 }
 
 .emptycontent {
-	margin-top: 0px;
+	margin-top: 24px;
 	margin-bottom: 10px;
-}
-.emptycontent > h2 {
-	padding-top: 20px;
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
```

Comment:
Out of curiosity: Are those changes required for the PR? They seem to be unrelated.

Reply:
That's right! I just wanted to vertically center the label "Loading messages"